infra/k3s/install.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Deploy a vanilla K3s cluster
+# Flannel CNI, network policy, kube-proxy and Traefik are disabled
+# to allow Cilium to handle all these
+echo 'Installing K3s...'
+
+curl -sfL https://get.k3s.io | sh -s - \
+  --flannel-backend none \
+  --disable-network-policy \
+  --disable-kube-proxy \
+  --disable=traefik \
+  --cluster-init
+
+# Copy kube config to home directory to access the cluster without sudo
+mkdir -p ~/.kube; \
+  sudo cat /etc/rancher/k3s/k3s.yaml > ~/.kube/config; \
+  echo 'export KUBECONFIG=~/.kube/config' >> ~/.bashrc; \
+  export KUBECONFIG=~/.kube/config
+
+# Setup architecture
+ARCH=amd64; if [ "$(uname -m)" = "aarch64" ]; then ARCH=arm64; fi
```

Comment:
```suggestion
ARCH=amd64
if [ "$(uname -m)" = "aarch64" ]; then 
  ARCH=x86_64
fi
```

Is this what we actually want?